vnext/ReactWindows.sln

```diff
@@ -289,6 +293,14 @@ Global
 		{700A84FD-F92A-43F1-8D06-B0E0745DF9B5}.Release|x64.Build.0 = Release|x64
 		{700A84FD-F92A-43F1-8D06-B0E0745DF9B5}.Release|x86.ActiveCfg = Release|Win32
 		{700A84FD-F92A-43F1-8D06-B0E0745DF9B5}.Release|x86.Build.0 = Release|Win32
+		{6F354505-FE3A-4BD2-A9A6-D12BBF37A85C}.Debug|ARM.ActiveCfg = Debug|x64
```

Comment:
>ARM [](start = 47, length = 3)

do we really need arm? who builds it?

Reply:
The x86.ActiveCfg and ARM.ActiveCfg are needed for the Visual Studio configuration manager UI. Even if I removed them, Visual Studio would add them back every time it opens the sln file. Since there are no Build.0 specifications for x86 and ARM, JSI.Desktop.UnitTests vcxproj is not built for those platforms.